go-client/build.gradle

```diff
@@ -0,0 +1,81 @@
+import io.deephaven.tools.docker.DiffTask
+
+plugins {
+    id 'com.bmuschko.docker-remote-api'
+    id 'io.deephaven.project.register'
+    id 'io.deephaven.deephaven-in-docker'
+}
+
+//TODO produce a registryProject
+//evaluationDependsOn Docker.registryProject('go')
+
+
+// The Go convention is to check in generated sources - while there is a "central repo server" for Go, it is a proxy
+// that merely polls each proxy several times an hour, but doesn't allow contents to be deployed to it. As such,
+// any generated code must exist in git, and will be downloaded from there. As part of the build then, we generate and
+// compare/overwrite protobuf sources.
+configurations {
+    go {}
+}
+dependencies {
+    go project(path: ':proto:proto-backplane-grpc', configuration: 'go')
+}
+def compare = tasks.register('compareProtobuf', DiffTask) {
+    expectedContents.set configurations.go
+    actualContents.set layout.projectDirectory
+    generateTask.set ':go-client:updateProtobuf'
+
+    ignore {
+        exclude 'internal/proto/**'
+    }
+}
+// fail a "check" build if these are out of date
+tasks.getByName('quick').dependsOn(compare)
```

Comment:
I know we use this elsewhere, but we should prefer to start moving to `tasks.named(...)` where applicable. New stuff prefer to follow that.

Reply:
ignoring these, as per slack discussion.